source/whats-new.txt

```diff
@@ -42,6 +42,10 @@ and features:
       the :ref:`MONGODB-OIDC: Kubernetes <kotlin-sync-auth-kubernetes>`
       section of the Enterprise Authentication Mechanisms guide
 
+   .. replacement:: atlas-query-operators
+
+      the `SearchOperator Interface API documentation <{+api+}/com/mongodb/client/model/search/SearchOperator.html>`__
```

Comment:
S: there's no search page but there is a section on the Aggregation page. Consider adding a note there?